pandas-stubs/core/groupby/generic.pyi

```diff
@@ -293,8 +329,22 @@ class DataFrameGroupBy(GroupBy):
         **kwargs,
     ) -> Series: ...
     def std(self, ddof: int = ...) -> DataFrame: ...
-    def sum(self, **kwargs) -> DataFrame: ...
+    def sum(
+        self,
+        numeric_only: bool = ...,
+        min_count: int = ...,
+        engine=...,
+        engine_kwargs=...,
+    ) -> DataFrame: ...
     def tail(self, n: int = ...) -> DataFrame: ...
     def take(self, indices: Sequence, axis: AxisType = ..., **kwargs) -> DataFrame: ...
     def tshift(self, periods: int, freq=..., axis: AxisType = ...) -> DataFrame: ...
     def var(self, ddof: int = ...) -> DataFrame: ...
+    def value_counts(
+        self,
+        subset: ListLike | None = ...,
+        normalize: bool = ...,
+        sort: bool = ...,
+        ascending: bool = ...,
+        dropna: bool = ...,
+    ) -> Series: ...
```

Comment:
I think you can do an `overload` here where it returns `Series[int]` if `normalize==False` (the default), and `Series[float]` if `normalize==True`